public/index.html

```diff
@@ -55,7 +55,7 @@
         <div class="container">
           <div class="row">
             <div class="col-s-4">
-              <h1 ng-click="market.selectTab('market')" class="push-left">{{market.title}}</h1>
+              <h1 ng-click="market.selectTab('market')" class="push-left">{{title}}</h1>
```

Comment:
selectTab('market') is a scope function? 
if so maybe you would like to do this:`ng-click=".selectTab('market')"`

and in the controller you'll need to do this:

```
$scope.selectTab= function (param) {

}
```